src/easyscore.ts

```diff
@@ -175,7 +175,7 @@ export class EasyScoreGrammar implements Grammar {
     return { token: '[0-9whq]+' };
   }
   TYPES(): Rule {
-    return { token: '[rRsSmMhHgG]' };
+    return { token: '[rRsSmMhHgGpP]' };
```

Comment:
Let's deal with this in a separate PR. Based on the changes below, we might need to call `build()` explicitly.

Reply:
pP removed